src/Data/Report.cs

```diff
@@ -37,28 +35,25 @@ public class Report
 
         public QueueThroughput[] Queues { get; init; }
 
+        [JsonIgnore(Condition = JsonIgnoreCondition.Never)] // Must be serialized even if 0 to maintain compatibility with old report signatures
         public long TotalThroughput { get; init; }
 
+        [JsonIgnore(Condition = JsonIgnoreCondition.Never)] // Must be serialized even if 0 to maintain compatibility with old report signatures
```

Comment:
Given the capabilities of System.Text.Json, seems it may make more sense to only decorate the two old properties that always _must_ be serialized for the sake of signature validation. Any new property we add that has a null/default value will then "just work" without decoration.